tests/test_geometry.py

```diff
@@ -388,3 +388,5 @@ def test_KLExpansion_projection():
     # Check that the projection is accurate
     rel_err = np.linalg.norm(signal-signal_proj)/np.linalg.norm(signal)
     assert np.isclose(rel_err, 0.0, atol=1e-5)
+
+# TODO. Add tests for _DefaultGeometry2D and add tests for default initialiations
```

Comment:
This can be added as an issue: "Add tests for _DefaultGeometry2D and add tests for default initializations"

Reply:
Added a basic test